rsmtool/notebooks/evaluation.ipynb

```diff
@@ -71,7 +71,9 @@
    "source": [
     "#### Association statistics\n",
     "\n",
-    "The table shows the standard association metrics between human scores and machine scores. Note that for the computation of kappas scores are always rounded.\n"
+    "The table shows the standard association metrics between human scores and machine scores. \n",
+    "\n",
+    "*Please note that if your data contained floating point scores, they are rounded for the computation of `kappa` and `wtkappa`.*\n"
```

Comment:
And agreement?

Reply:
Again, I will defer to you about what you want to say here.